README.md

```diff
@@ -39,45 +42,46 @@ To run the integration tests or Python scripts (not required to use the code), i
 Python (3.7 or higher), and install python3 packages using `./scripts/pysync` (requires poetry) in the project directory.
 
 2. Build neon and patched postgres
+
```

Comment:
Some of the changes are because of my editor auto-formatting, which I think, are good to keep.

Reply:
`make fmt` runs `rustfmt` and `yapf`, it won't affect markdown files. We don't have a consistent format for markdown files. For example, I line-wrap the paragraphs with newlines, but others put whole paragraph on a single line. Perhaps we should run all the markdown through a formatter at some point, for the sake of consistency, but it doesn't matter much. Until we do, "follow the surrounding style" is a good general rule.

(The extra newlines you're adding to the README here seem fine to me. I don't think they would be overridden by anyone else's formatter)